webpack/assets/javascripts/react_app/components/common/dates/dates.stories.js

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { date, boolean, select, withKnobs } from '@storybook/addon-knobs';
+import IsoDate from './IsoDate';
+import LongDateTime from './LongDateTime';
+import RelativeDateTime from './RelativeDateTime';
+import ShortDateTime from './ShortDateTime';
+import { i18nProviderWrapperFactory } from '../../../common/i18nProviderWrapperFactory';
+import Code from '../../../../../../stories/components/Code';
+import Story from '../../../../../../stories/components/Story';
+import Text from '../../../../../../stories/components/Text';
+
+storiesOf('Components/Common', module)
+  .addDecorator(withKnobs)
+  .add('Dates', () => {
+    const now = new Date();
+    const defaultValue = new Date('2018-11-12 00:54:55 -1100');
+
+    const dateToShow = new Date(
+      date('Date and time in your time zone', defaultValue)
+    );
+    const showSeconds = boolean('Show seconds');
+
+    const timezoneOptions = [
+      'America/Phoenix',
+      'America/Chicago',
+      'America/New_York',
+      'UTC',
+      'Europe/Prague',
+      'Europe/Kiev',
+      'Asia/Tel_Aviv',
```

Comment:
its ```Asia/Jerusalem``` AFAIR :)

Reply:
Ok, I can change that

Reply:
It worked for me. I didn't notice `Asia/Tel_Aviv` tz is deemed to be deprecated, but obviously it's still available for backwards compatibility. Anyway a changed it to Jerusalem.